detectors/python-pickle-insecure-deserialization/python-pickle-insecure-deserialization_compliant.py

```
@@ -0,0 +1,13 @@
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#  SPDX-License-Identifier: MIT-0
+
+# {fact rule=python-pickle-insecure-deserialization@v1.0 defects=0}
+import pickle
+import requests
+
+def positive_dumps_loads():
+    data = bytes(requests.get("https://data.bytes"))
```

Ruff (0.13.1)

Probable use of `requests` call without timeout

(S113)